docs/pages/access-controls/guides/dual-authz.mdx

```diff
@@ -159,19 +202,19 @@ spec:
       'type': 'db'
 ```
 
-Example below creates local users Alice, Ivan and Bob.
-
-Bob does not have a role `dbadmin` assigned to him, but can create an access request for it.
+The commands below create the local users Alice, Ivan and Bob.
 
 ```code
 $ tctl users add bob@example.com --roles=devops
 $ tctl users add alice@example.com --roles=reviewer
 $ tctl users add ivan@example.com --roles=reviewer
 ```
 
-## Access Requests flow
+### Create an Access Request
+
+Bob does not have a role `dbadmin` assigned to him, but can create an Access Aequest for it.
 
-Bob can create an access request for the `dbadmin` role in the Web UI or CLI:
+Bob can create an Access Request for the `dbadmin` role in the Web UI or CLI:
```

Comment:
Does it make sense to capitalize "access request"?

Reply:
I see it both capitalized and lowercased throughout the docs. Thinking about it again, I'm not sure it's something we want to make a product name, since it also works as a generic term. I can lowercase it here.